File: Python/378A-playingwithdice.py
a, b = map(int, input().split())
awins = 0
bwins = 0
draws = 0
for i in range(1, 7):
    if abs(a - i) > abs(b-i):
        bwins += 1
    elif abs(a - i) == abs(b - i):
        draws += 1
    else:
        awins += 1
print(awins, draws, bwins)

File: C++/A/378A-playingwithdice.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int a, b, awins = 0, bwins = 0, draws = 0;
    cin >> a >> b;
    for (int i = 1; i < 7; i++)
    {
        if (abs(a - i) > abs(b - i))
            bwins++;
        else if (abs(a - i) == abs(b - i))
            draws++;
        else
            awins++;
    }
    cout << awins << ' ' << draws << ' ' << bwins;
}